WineCellar/WineCellar.ControllerTest/ModelWineControllerShould.cs

```diff
@@ -51,8 +51,8 @@ public async Task WineController_FilterShould()
             var notes = new List<string>();
             notes.Add("sweet");
             
-            var filteredList = Data.FilterWine(wineList, "Test", 150, 250, "TESTTYPE", "A1.1", 1999, 2001, notes, 5);
-            Assert.NotNull(filteredList);
+            //var filteredList = Data.FilterWine(wineList, "Test", 150, 250, "TESTTYPE", "A1.1", 1999, 2001, notes, 5);
```

Comment:
Zijn de comments nodig?

Reply:
Ja, deze functie bestaat niet meer. Deze is opgesplitst in meerdere functies. Hier zouden nog aparte tests voor geschreven moeten worden.